server/src/requirements.txt

```diff
@@ -0,0 +1,7 @@
+fastapi==0.61.1
+uvicorn>=0.11.1
+python-multipart
+
+# dev
+pytest>=5.3.2
+requests
```

Comment:
I suggesting moving this to a `requirements-dev.txt`

Reply:
But they also need to be removed from this `requirements.txt` :sweat_smile:

Reply:
ahah yes I know (I will remove once we are okay with everything else, it let me change nothing to run the tests locally :D )